modules/swift/examples/code_snippets/SampleCodeTest.swift

```diff
@@ -1209,2824 +1208,1603 @@ class SampleCodeTest {
-                    ) // <.>
-
-                // ALTERNATIVELY unpack in steps
-                this_hotel.id = thisJsonObj["id"] as! String
-                this_hotel.name = thisJsonObj["name"] as! String
-                this_hotel.type = thisJsonObj["type"] as! String
-                this_hotel.city = thisJsonObj["city"] as! String
-                hotels[this_hotel.id] = this_hotel
-
-
-            } // end for
-
-            // end::query-access-json[]
-
-        } catch let err {
-            print(err.localizedDescription)
-
-        } // end do
-
-
-
-    } // end func dontTestQueryAll
-
-
-
-    func dontTestQueryProps () throws {
+    
+    func dontTestQuerySyntaxProps() throws {
         // tag::query-syntax-props[]
-        let db = try! Database(name: "hotel")
-        var hotels = [String:Any]()
-        var hotel:Hotel = Hotel.init()
-
-        let listQuery = QueryBuilder
+        let database = try! Database(name: "hotel")
+        
+        let query = QueryBuilder
             .select(SelectResult.expression(Meta.id).as("metaId"),
                     SelectResult.expression(Expression.property("id")),
                     SelectResult.expression(Expression.property("name")),
                     SelectResult.expression(Expression.property("city")),
                     SelectResult.expression(Expression.property("type")))
-                    .from(DataSource.database(db))
-
+            .from(DataSource.database(database))
+        
         // end::query-syntax-props[]
-
+        print(query)
+    }
+    
+    func dontTestQueryAccessProps () throws {
+        let query = QueryBuilder.select(SelectResult.all()).from(DataSource.database(database))
+        var hotels = [String: Hotel]()
+        
         // tag::query-access-props[]
-        for (_, result) in try! listQuery.execute().enumerated() {
-
-
-            let thisDoc = result.toDictionary() as? [String:Any]  // <.>
-                // Store dictionary data in hotel object and save in arry
-            hotel.id = thisDoc!["id"] as! String
-            hotel.name = thisDoc!["name"] as! String
-            hotel.city = thisDoc!["city"] as! String
-            hotel.type = thisDoc!["type"] as! String
-            hotels[hotel.id] = hotel
-
+        for result in try! query.execute() {
+            
+            
+            let doc = result.toDictionary()  // <.>
+            
+            // Store dictionary data in hotel object and save in arry
+            guard let id = doc["id"] as? String else {
+                continue
+            }
+            var hotel = Hotel(id: id)
+            hotel.name = doc["name"] as? String
+            hotel.city = doc["city"] as? String
+            hotel.type = doc["type"] as? String
+            hotels[id] = hotel
+            
             // Use result content directly
-            let docid = result.string(forKey: "metaId")
+            let docid = result.string(forKey: "metaId") // Selected (Meta.id).as("metaId")
```

Comment:
There is no metaId in the query.

Reply:
there is metaId in the above sample 'dontTestQuerySyntaxProps' 
`tag::query-syntax-props[]`